paywall/src/data-iframe/cache/localStorage.js

```diff
@@ -43,6 +45,42 @@ function getContainer(window, key, type = false) {
   return container
 }
 
+/**
+ * Retrieve a cached value for a non-user-specific type on a specific network
+ *
+ * @param {object} window this is the global context, either global, window, or self
+ * @param {int} networkId the ethereum network id
+ * @param {string} type the type of account data to retrieve
+ */
+export async function getReadOnly(window, networkId, type) {
```

Comment:
Is there some context around these methods ending with `ReadOnly`?

Reply:
caching read-write values (keys, transactions) depends on a user account. read-only values are currently locks, which are account-independent